x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/warnings_popover.scss

```diff
@@ -0,0 +1,15 @@
+.lnsWorkspaceWarning__button {
+  color: $euiColorWarningText;
+}
+
+.lnsWorkspaceWarningList {
+  overflow-y: auto;
```

Comment:
Please replace `overflow-y: auto;` with `@include euiYScroll;` to better match the designs and use EUI scrollbars.